src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -374,6 +375,12 @@ private void extractPageMetadata(final String... aRow) throws CsvParsingExceptio
                         }
                         break;
                     }
+                    case "audio": {
```

Comment:
I wonder if we should also check here whether width and height are present, and throw an error if so. Thoughts?

Reply:
Seems like a "nice to have"... This might also fall into the "strict API" vs. "loose API" argument (e.g., https://news.ycombinator.com/item?id=24918927). In this case, since we're taking in user data from a CSV, one might argue to be forgiving and just ignore what is invalid (as long as it is ignorable and not something that's going to cause a problem with the processing). Or... maybe it's an indication that something is wrong and should be flagged for further investigation.

Reply:
Seems some more debate needed here...

Reply:
I think I do feel pretty strongly about throwing an error if there's w/h on audio. As a user, this is something I imagine I would want to know and correct.

Reply:
👍🏼

Reply:
Added check for populated height/width in audio to CsvParser, and a test case/resource to verify

Reply:
Looks great, thanks!